config/etc/zinit/init/ntpd.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+ntp_flags=$(grep -o 'ntp=.*' /proc/cmdline | sed 's/^ntp=//')
+
+params=""
+if [ -n "$ntp_flags" ]; then
+  params=$(echo "-p $ntp_flags" | sed s/,/' -p '/g)
```

Comment:
I am wondering if people can do something like 
```
ntp="\" && echo 'key' > /root/.ssh/authorized_keys && exit 0"
```
to inject his key into ssh authorized keys via command line.